proxy/http/Http1ClientSession.cc

```diff
@@ -465,6 +465,12 @@ Http1ClientSession::new_transaction()
     return;
   }
 
+  if (!client_vc->add_to_active_queue()) {
```

Comment:
I would break up default timeout and active queue changes into different PRs.

Reply:
Makes sense. Will split it.

Reply:
https://github.com/apache/trafficserver/pull/6755